src/components/Button/Button.test.tsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react'
+import { render } from '@testing-library/react'
+
+import Button from './Button'
+
+describe('Button', () => {
+  const  { container } = render(<Button>Найти</Button>)
+  
+  describe('common', () => {
+    const component = container.querySelector('.Button')
+    
+    it('renders correctly', () => {
+      expect(component).toHaveTextContent('Найти')
+    })
+
+    it('matches snapshot', () => {
+      expect(component).toMatchSnapshot()
+    })
+  })
+
+  describe('with extra className and disabled', () => {
+    const  { container } = render(<Button className={this.className} disabled={true}>Найти</Button>)
+  
+    const component = container.querySelector('.Button.Button_disabled')
+
+    it('renders correctly', () => {
+      expect(component).toHaveTextContent('Найти')
```

Comment:
Переименуй для разнообразия:
```suggestion
      expect(component).toHaveTextContent('Сохранить')
```